src/constant.ts

```diff
@@ -15,6 +15,7 @@ export const WHITELISTED_ORIGINS = [
   '*.coinbase.com',
   '*.hcaptcha.com',
   '*.robinhood.com',
+  '*.google.com',
```

Comment:
@balint-front let's also add 'https://google.com' to the list so if there is covers subdomain and domain of google